index.js

```diff
@@ -235,8 +235,15 @@ function loadConfig(program) {
       }
 
       // merge into single var after all potential saveConfigs finished
-      config = {...config, ...parsedConfig}
-      config.tags = process.env.LOGDNA_TAGS || config.tags
+      config = {
+        ...config
+      , ...parsedConfig
+      , tags: process.env.LOGDNA_TAGS || config.tags
+      , flushTimer: null
+      , healthcheckTimer: null
+      , rescanTimer: null
```

Comment:
We need to store them somewhere so that they can be canceled.  As a general style, objects need to be pre-defined with all properties to avoid map transitions.  Plus, adding these things to `config` on-the-fly gets them lost in code.  So, we add them here 1) to be clear that they will be defined and 2) so that we can cancel them in shutdown.